lms/djangoapps/ccx/plugins.py

```diff
@@ -28,5 +29,7 @@ def is_enabled(cls, course, user=None):
             return True
```

Comment:
Question: Why does this return true if user is None? Can we update the docs to reflect this case?